### PR TITLE
#84 - new attempt to update vignette for adce

### PR DESCRIPTION
**Pull request by iaugu**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bms63** [comment]
@iaugu Looing better? 

You can run the jobs (not sure why they were cancelled) by clicking on Details and then Clicking 

image

image

**Comment by iaugu** [comment]
I hope this is all fine now, if not, let me know what actions I should do to correct.

**Comment by bms63** [comment]
Everything is looking good! @federicobaratin  and @YaminiBollu  can you review this PR for @iaugu.  I think this is the next step for you all, i.e. reviewing Pull Requests.

**Comment by YaminiBollu** [comment]
> Everything is looking good! @federicobaratin and @YaminiBollu can you review this PR for @iaugu. I think this is the next step for you all, i.e. reviewing Pull Requests.

All checks are passed ..super good.. 👍 just a minor thing i noticed when going through Rmd file. may be we can remove "vaccine" in the documentation part. As we have reactogenicity data in pharma as well and i feel like even they can adapt this template for their studies.. please comment.. 
adce_vignette

**Comment by bms63** [comment]
> > Everything is looking good! @federicobaratin and @YaminiBollu can you review this PR for @iaugu. I think this is the next step for you all, i.e. reviewing Pull Requests.
> 
> All checks are passed ..super good.. 👍 just a minor thing i noticed when going through Rmd file. may be we can remove "vaccine" in the documentation part. As we have reactogenicity data in pharma as well and i feel like even they can adapt this template for their studies.. please comment.. adce_vignette

Good point @YaminiBollu !  I think since the scope of the package is for Vaccines, then we can leave as is for now.  However, this brings up some past struggles `admiralonco` had with `admiral`, i.e. where should functions live to make it more accessible.  I'll make an issue of this for later discussion

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ date: '2022-06-27'
 
 <!-- Please do not edit the README.md file as it is auto-generated. Only edit the README.Rmd file -->
 
-# Admiral Extension for Vaccines <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
+# admiral package extension Template <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
```
Comment (bms63) [inline README.md:11]:
```suggestion
# Admiral Extension for Vaccines <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
```

File: vignettes/adce.Rmd [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      relative_url,
+      sep = "/"
+    )
+  }
+  return(link(text, url))
+}
```
Comment (bms63) [inline vignettes/adce.Rmd:43]: do you mind removing this?  it is not needed for a vignette

Comment (larmisha22) [reply vignettes/adce.Rmd]: This has been removed.

**Review by kbodicherla — APPROVED** [review]
Review done

**Pull request merged**